src/main/java/org/broadinstitute/hellbender/utils/Dirichlet.java

```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.hellbender.utils;
+
+import org.apache.commons.math3.special.Gamma;
+import org.broadinstitute.hellbender.utils.param.ParamUtils;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.stream.IntStream;
+
+/**
+ * The Dirichlet distribution is a distribution on multinomial distributions: if pi is a vector of positive multinomial weights
+ * such that sum_i pi[i] = 1, the Dirichlet pdf is P(pi) = [prod_i Gamma(alpha[i]) / Gamma(sum_i alpha[i])] * prod_i pi[i]^(alpha[i] - 1)
+ *
+ * The vector alpha comprises the sufficient statistics for the Dirichlet distribution.
+ *
+ * Since the Dirichlet is the conjugate prior to the multinomial, if one has a Dirichlet prior with concentration alpha
+ * and observes each category i n_i times (assuming categories are drawn from a multinomial distribution pi)
+ * the posterior is alpha_i -> alpha_i + n_i
+ *
+ *
+ * @author David Benjamin &lt;davidben@broadinstitute.org&gt;
+ */
+public class Dirichlet {
```

Comment:
i think this class could use a bit of javadoc for the constructors and non-trivial methods, and should be covered by some simple unit tests.

also, perhaps the name should be `DirichletDistribution`?  although right now there's essentially just one method, so it's a bit misleading as this doesn't implement any of the common abstract distribution interfaces.  (could refactor so the method is static, but i think it is nicer as is.)